STM32Cube/Tasks/controller.c

```diff
@@ -7,10 +7,12 @@
 
 #include "controller.h"
 #include "freertos.h"
+#include "queue.h"
 #include "cmsis_os.h" //TODO replace with xtaskDelayUntil
 #include "stm32h7xx_hal.h"
 #include <math.h>
 #include "millis.h"
+#include "flight_phase.h"
```

Comment:
Either include can_handler.h or extern busQueue (probably better to include right?)